src/pages/transactions/create/Create.tsx

```diff
@@ -0,0 +1,209 @@
+        base_type: transaction?.base_type === 'deposit' ? 'CREDIT' : 'DEBIT',
+      });
+      toast.success(t('created_transaction'), { id: toastId });
+      setIsSaving(false);
+      navigate(route('/transactions'));
+    } catch (cachedError) {
+      setIsSaving(false);
+      const error = cachedError as AxiosError;
+      console.error(error);
+
+      if (error?.response?.status === 422) {
+        setErrors(error?.response?.data?.errors);
+        toast.dismiss();
+      } else {
+        toast.error(t('error_title'));
+      }
+    }
+  };
+
+  useEffect(() => {
+    setTransaction((prevState) => ({
+      ...prevState,
+      base_type: 'deposit',
+      currency_id: company?.settings?.currency_id,
+      date: date(new Date().toString(), 'YYYY-MM-DD'),
+      amount: 0,
+    }));
+    setCurrencySeparators(getCurrencySeparators(currencies[0]?.id));
+  }, [currencies]);
+
+  return (
+    <Default
+      title={documentTitle}
+      breadcrumbs={pages}
+      onBackClick="/transactions"
+    >
+      <Container>
+        <Card
+          title={documentTitle}
+          withSaveButton
+          onSaveClick={onSave}
+          disableSubmitButton={isSaving}
+        >
+          <Element required leftSide={t('type')}>
+            <SelectField
+              value={transaction?.base_type}
+              onValueChange={(value) => handleChange('base_type', value)}
+              errorMessage={errors?.base_type}
+            >
+              {Object.values(transactionTypes).map((transactionType) => (
+                <option key={transactionType} value={transactionType}>
+                  {t(`${transactionType}`)}
+                </option>
+              ))}
+            </SelectField>
+          </Element>
+          <Element required leftSide={t('date')}>
+            <InputField
+              type="date"
+              value={transaction?.date}
+              onValueChange={(value) => handleChange('date', value)}
+              errorMessage={errors?.date}
+            />
+          </Element>
+          <Element required leftSide={t('amount')}>
+            <DecimalNumberInput
+              border
+              precision={currencySeparators?.precision}
+              currency={currencySeparators}
+              className="auto"
+              initialValue={transaction?.amount?.toString()}
+              value={transaction?.amount?.toString()}
+              onChange={(value: string) => handleChange('amount', value)}
+              errorMessage={errors?.amount}
+            />
+          </Element>
+          <Element required leftSide={t('currency')}>
+            <SelectField
+              value={transaction?.currency_id}
+              onValueChange={(value) => handleChange('currency_id', value)}
+              errorMessage={errors?.currency_id}
+            >
+              {currencies?.map(({ id, name }) => (
+                <option key={id} value={id}>
+                  {t(`${name}`)}
```

Comment:
No need to escape, this is valid JSX.

`{t(name)}`

Reply:
> No need to escape, this is valid JSX.
> 
> `{t(name)}`

changed